source/locale/pt_BR/LC_MESSAGES/using-api-interface.po

```diff
@@ -274,122 +263,132 @@ msgstr "Se não houver saída (output), a mensagem foi enviada ao *broker* MQTT.
-"id>/attrs`` (e.g.: ``admin:efac/attrs``)."
+"The topic to publish messages has the pattern ``<tenant>:<device-id>/attrs`` (e."
+"g.: ``admin:efac/attrs``)."
 msgstr ""
-"O tópico para publicação deve assumir o padrão ``<tenant>:<device-"
-"id>/attrs`` (por exemplo: ``admin:efac/attrs``)."
+"O tópico para publicação deve assumir o padrão ``<tenant>:<device-id>/attrs`` "
+"(por exemplo: ``admin:efac/attrs``)."
 
 #: ../../source/using-api-interface.rst:301
 msgid ""
-"The topic to subscribe should has the pattern ``<tenant>:<device-"
-"id>/config`` (e.g.: ``admin:efac/config``)."
+"The topic to subscribe should has the pattern ``<tenant>:<device-id>/config`` "
+"(e.g.: ``admin:efac/config``)."
 msgstr ""
-"O tópico de subscrição deve assumir o padrão ``<tenant>:<device-"
-"id>/config`` (por exemplo: ``admin:efac/config``)."
+"O tópico de subscrição deve assumir o padrão ``<tenant>:<device-id>/config`` "
+"(por exemplo: ``admin:efac/config``)."
 
 #: ../../source/using-api-interface.rst:304
 #: ../../source/using-api-interface.rst:359
 msgid ""
-"MQTT payload must be a JSON with each key being an attribute of the dojot"
-" device, such as:"
+"MQTT payload must be a JSON with each key being an attribute of the dojot "
+"device, such as:"
 msgstr ""
-"Os dados da mensagem MQTT (payload) devem ser um JSON com cada chave "
-"sendo um atributo do dispositivo cadastrado na dojot, como:"
+"Os dados da mensagem MQTT (payload) devem ser um JSON com cada chave sendo um "
+"atributo do dispositivo cadastrado na dojot, como:"
 
 #: ../../source/using-api-interface.rst:311
 msgid "Using Mosca (legacy)"
 msgstr "Usando o Mosca (legado)"
 
 #: ../../source/using-api-interface.rst:314
 msgid ""
-"VerneMQ is the new default MQTT broker. Support for Mosca will be "
-"eventually dropped, so use VerneMQ if possible!"
+"VerneMQ is the new default MQTT broker. Support for Mosca will be eventually "
+"dropped, so use VerneMQ if possible!"
 msgstr ""
-"VerneMQ é o novo broker MQTT padrão. O suporte ao Mosca será "
-"eventualmente retirado, então use o VerneMQ se possível!"
+"VerneMQ é o novo broker MQTT padrão. O suporte ao Mosca será eventualmente "
+"retirado, então use o VerneMQ se possível!"
+
+#: ../../source/using-api-interface.rst:317
+msgid "Let's send a message to dojot:"
+msgstr "Vamos publicar a mensagem a seguir:"
+
+#: ../../source/using-api-interface.rst:327
+msgid ""
+"**Also you can send a configuration message from dojot to the device to change "
+"some of its attributes. The target attribute must be of type “actuator”.**"
+msgstr ""
+"**Além disso, você pode enviar uma mensagem de configuração da dojot para o "
+"dispositivo. O atributo de destino deve ser do tipo “actuator” ou “atuador”.**"
+
+#: ../../source/using-api-interface.rst:336
+msgid "Triggering the sending of the message from the dojot to the device."
+msgstr "Disparando a publicação da mensagem da dojot para o dispositivo."
```

Comment:
Disparando envio de mensagens da

Reply:
Changed for "Disparando envio de mensagens da"